docs/user_guide/specifying_finance_parameters.md

```diff
@@ -173,3 +166,61 @@ params:
   tax losses monetized: True
   loan period if used: 0
 ```
+
+
+# Cost year of Cost Models
+Some cost models are derived from literature and output costs (CapEx and OpEx) in a specific dollar-year. Some cost models require users to input the key cost information, and the output costs are in the same cost year as the user-provided costs. For [cost models with a built-in cost year](#cost-models-with-inherent-cost-year), the cost year is not required as an input for the cost model. For [cost models based on user provided costs](#cost-models-with-user-input-cost-year), the `cost_year` should be included in the tech_config for that technology.
+
+## Cost models with inherent cost year
+
+### Summary of cost models that are based around a cost year
+| Cost Model              | Cost Year  |
+| :---------------------- | :---------------: |
+| `basic_electrolyzer_cost`|  2016    |
+| `pem_electrolyzer_cost`|  2021    |
+| `singlitico_electrolyzer_cost`|  2021    |
+| `h2_storage`  with `'mch'` storage type  |  2024    |
+| `h2_storage` for geologic storage or buried pipe | ? 2018 ? |
+| `simple_ammonia_cost`   |  2022    |
+| `direct_ocean_capture_cost` | 2023 |
+| `steel_cost`            |  2022    |
+| `reverse_osmosis_desalination_cost` | ? 2013 ? |
```

Comment:
For these technologies with the years in question, could you please either determine the correct year? If that's too arduous, we should make it clear we're not certain on these years within the doc with a short sentence.